frozen_requirements.txt

```diff
@@ -85,7 +85,7 @@ pyasn1==0.4.4
 pyasn1-modules==0.2.2
 pycodestyle==2.3.1
 pycparser==2.19
-pycrypto==2.6.1
+pycryptodome==3.7.0
```

Comment:
Would this not require changes to `test_requirements.txt` too?

Reply:
I'm not fully clear on how the security vulnerabilities apply to test_requirements.txt. test_requirements.txt doesn't have any mentions of installing via the base-image, specifically python3-crypto. I could use help here if I'm missing something.

Reply:
I mean that we currently have two different "requirements" files that could now be out of sync due to the switch from `pycrypto` to `pycryptodome`. (but looking at the file there isn't anything specified there)